ROADMAP.md

```diff
@@ -87,6 +87,103 @@ We think that it should be done the other way around, i.e. providing a low-level
 
 On the other hand, using a smart date library for the DatePicker / TimePicker would probably be much better as date management is tricky and not a core business.
 
+## Breaking changes before v1
+
+It's time to look at the last breaking changes needed before releasing Material-UI v1.
+Users trying out and using v1-beta and giving feedback has been a tremendous help. Thank you!
+
+This feedback has guided the following list of important breaking changes
+that **are needed for the stable version**:
+
+- [ ] Rename `Reboot` to `CssBaseline` [#10233](https://github.com/mui-org/material-ui/issues/10233).
+  The new wording should clarify the purpose of the component. For instance, it's not about adding JavaScript polyfills.
+
+   ```diff
+   -<Reboot />
+   +<CssBaseline />
+   ```
+
+- [ ] Remove the `fontSize` property from the `Icon` and `SvgIcon` components in order to make it the default behavior.
+It's already the default behavior of the `Icon` component. You will still be able to change the size of the icons with the `width` and `height` CSS properties. The difference is that they can use the `font-size` as a shorthand.
+
+  ```diff
+  -<SvgIcon style={{ fontSize: 20 }} fontSize />
+  +<SvgIcon style={{ fontSize: 20 }} />
+  ```
+
+- [ ] Do not prefix the classes keys when used for a variation. >80% of the components enforce this rule, let's make this figure 100%.
+
+  ```diff
+  -<TableCell variant="head" classes={{ typeHead: 'typeHead' }} />
+  +<TableCell variant="head" classes={{ head: 'head' }} />
+  ```
+
+- [ ] Remove the `xxxClassName` properties when already covered the `classes` property.
+  These properties were introduced before`classes`.
+  Using a single pattern makes things more predictable and easier to work with.
+
+  ```diff
+  -<Tabs buttonClassName="buttonClassName" indicatorClassName="indicatorClassName" />
+  +<Tabs classes={{ buttonClassName: 'buttonClassName', indicatorClassName: 'indicatorClassName' }} />
+  ```
+
+- [ ] Grid with no spacing by default [#10223](https://github.com/mui-org/material-ui/issues/10223).
+  The negative margin implementation solution currently used comes with [serious limitations](https://material-ui-next.com/layout/grid/#negative-margin).
+  Material-UI is the only library with a non-zero default spacing between the items.
+  Having zero spacing by default will ease the usage of the component.
+
+  ```diff
+  -<Grid />
+  +<Grid spacing={8} />
+  ```
+
+- [ ] Flatten the import path [#9532](https://github.com/mui-org/material-ui/issues/9532).
+  Knowing the component name should be enough for being able to import it.
+
+  ```diff
+  -import CircularProgress from 'material-ui/Progress/CircularProgress';
+  +import CircularProgress from 'material-ui/CircularProgress';
```

Comment:
It might also be helpful to show an example of a component with subcomponents.

Reply:
(Also scoped package name?)

Reply:
Yeah. I will add a new bullet point for this.